setup.py

```diff
@@ -91,7 +91,6 @@ def get_package_data(package):
     packages=get_packages(package),
     package_data=get_package_data(package),
     cmdclass={'test': PyTest},
-    install_requires=install_requires,
```

Comment:
@doctoryes if you're removing this you should also remove the line above that assigns `install_requires`. Out of curiosity, why aren't those changes already on this branch?